DEV_SETUP.md

```diff
@@ -198,7 +198,7 @@ Save those backups to somewhere you'll be able to access from the new environmen
     sudo pip install virtualenvwrapper --ignore-installed six
     ```
 
-- For downloading Python 3.6 consider:
+- For downloading Python 3.9 consider:
 
   - Using [pyenv](https://github.com/pyenv/pyenv-installer)
   - Using Homebrew with this [brew formula](https://gist.github.com/SamuelMarks/0ceaaf6d3de12b6408e3e67aae80ae3b)
```

Comment:
@snopoke  It looks like this formula is for installing Python 3.6.8. Does this need to be replaced also?

Reply:
https://github.com/dimagi/commcare-hq/pull/30922